src/style.css

```diff
@@ -1,3 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.box_empty {
+  display: flex;
+  height: 60px;
+  flex-grow: 1;
+}
+
+.container {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
+  margin-left: 0;
+  margin-right: 50px;
+}
+
+.titles {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  list-style: none;
+  align-items: center;
+  display: flex;
+}
+
+.menu {
+  color: black;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+}
+
+.menu:hover {
+  text-decoration: underline;
+  text-decoration-thickness: 3px;
+  text-underline-offset: 0;
+  text-decoration-color: #00acdc;
+}
+
+.logo {
+  position: absolute;
+  z-index: 1;
+  top: 10px;
+  left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.menu-4:hover {
+  color: #00acdc;
+  text-decoration: none;
+}
+
+.a.is-active {
```

Comment:
The selector `.a.is-active` seems incorrect. It should be `a.is-active` without the dot before `a`, as you are targeting an `<a>` tag with the class `is-active`. Please correct this to ensure the styles are applied properly.